wfe/wfe.go

```diff
@@ -818,7 +818,7 @@ func (wfe *WebFrontEndImpl) KeyRollover(
 		wfe.sendError(acme.MalformedProblem("Inner JWS verification error"), response)
 		return
 	}
-	
```

Comment:
Sorry for adding the spaces!

Reply:
TBH I didn't even notice that I removed it in this branch! Might have been an accident :-)

Reply:
I think your editor is doing this, as you checked the same thing in in #160 then ;)